client/app/routes/timetable_.$year.$programme.$level.$sem.$day.tsx

```diff
@@ -115,7 +115,9 @@ export default function TimeTable() {
               <div className="size-4">
                 <div className="i-lucide-list-plus opacity-50" />
               </div>{" "}
+              <span className="text-blue-400">
```

Comment:
Any reason for this?

Reply:
No please it must have been a typo

Reply:
Dw I am going to close my pull request seems like I am causing too much problems , thanks for your time tho